indico/htdocs/sass/modules/_contributions.scss

```diff
@@ -21,3 +21,69 @@
 @import "contributions/layout";
 @import "contributions/display";
 @import "contributions/editor";
+
+
+@mixin item-picker-label() {
+    .label {
+        float: left;
+        max-width: 80%;
+        overflow: hidden !important;
+        text-overflow: ellipsis;
+        display: inline-block;
+        line-height: 1.2em;
+    }
+
+    .icon-arrow-down {
+        float: right;
+    }
+}
+
+.session-item-picker {
+    width: 100%;
+    line-height: 0.9em;
+    white-space: nowrap;
+
+    @include item-picker-label();
```

Comment:
It's good practice having mixins specified at the beginning rather than at the end.